src/main/java/werkit/UI.java

```diff
@@ -495,4 +504,200 @@ public void printClearedScheduleMessage() {
     public void printSearchNotFoundMessage(String category) {
         System.out.println("Sorry, no matching " + category + " found.");
     }
+
+    /**
+     * Prints help message for 'plan /new' command.
+     */
+    public void printPlanAddHelp() {
+        System.out.println("\t To add a plan, please enter: ");
+        printColorText(TextColor.COLOR_YELLOW, "\t plan /new <plan name> /workouts"
+                + " <workout index(s) separated by ','>");
+        System.out.println("\t Example: ");
+        printWorkoutExample();
+        printPlanAddExample();
+    }
+
+    /**
+     * Prints workout list examples.
+     */
+    public void printWorkoutExample() {
+        System.out.println("\t The workout list upon entering "
+                + getColorText(TextColor.COLOR_YELLOW, "workout /list")
+                + ", contains the\n \t following workouts:");
+        printColorText(TextColor.COLOR_YELLOW, "\t 1. push up (10 reps)");
+        printColorText(TextColor.COLOR_YELLOW, "\t 2. sit up (10 reps)");
+        printColorText(TextColor.COLOR_YELLOW, "\t 3. pull up (10 reps)");
+    }
+
+    /**
+     * Prints the example of adding plans.
+     */
+    public void printPlanAddExample() {
+        System.out.println("\t To create a new plan, enter the following command:");
+        printColorText(TextColor.COLOR_YELLOW, "\t plan /new Grow My Muscles /workouts 1, 2");
+        System.out.println("\t A new plan named " + getColorText(TextColor.COLOR_YELLOW, "Grow My Muscles")
+                + " with workout index " + getColorText(TextColor.COLOR_YELLOW, "1") + " and "
+                + getColorText(TextColor.COLOR_YELLOW, "2\n")
+                + "\t will be created and added to the application's list of plans.");
+    }
+
+    /**
+     * Prints the example plan list to be used in schedule update help message.
+     */
+    public void printPlanExample() {
+        System.out.println("\t The plan list upon entering "
+                + getColorText(TextColor.COLOR_YELLOW, "plan /list") + ", contains the\n \t following plans:");
+        printColorText(TextColor.COLOR_YELLOW, "\t 1. arms");
+        printColorText(TextColor.COLOR_YELLOW, "\t 2. stronger arms");
+        printColorText(TextColor.COLOR_YELLOW, "\t 3. Grow My Muscles");
+    }
+
+    /**
+     * Prints help message for 'plan /list' command.
+     */
+    public void printPlanListHelp() {
+        System.out.println("\t To view all plans, please enter:");
+        printColorText(TextColor.COLOR_YELLOW, "\t plan /list");
+        System.out.println("\t This will print all the existing plans.");
+    }
```

Comment:
don't forget to add in the plan /details to view a plan in more details once implemented !

Reply:
I have added this into the help message, thank you!